### PR TITLE
Update Edit Footer for Cards

### PR DESCRIPTION
**Pull request by zsarnett**
## Proposed change
Updates the Style of the Edit Options to better flow with the current Card. This fixes the Material Design issues as there is not Shadow footer

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [x] Code quality improvements to existing code or addition of tests

## Additional information

image

:arrow_down:

(old image with out HA-CARD) ❗️ 
image

(New Image with HA-CARD) ❗️ 
image

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.

### CODE REVIEW

**Review by balloob — APPROVED** [review]
So good!

File: src/panels/lovelace/components/hui-card-options.ts [hunk 1]
```diff
@@ -99,6 +99,10 @@ export class HuiCardOptions extends LitElement {
           rgba(0, 0, 0, 0.12) 0px 1px 5px -4px,
           rgba(0, 0, 0, 0.2) 0px 3px 1px -2px;
         display: flex;
+        position: relative;
+        margin-top: -1px;
+        border-radius: 0px 0px var(--ha-card-border-radius, 2px)
```
Comment (balloob) [inline src/panels/lovelace/components/hui-card-options.ts]: This works but now you're including defaults from somewhere else, that is bound to go out of sync when the defaults are updated . 

Instead just set `border-top-left-radius: 0` and `border-top-right-radius: 0`

**Review by balloob — APPROVED** [review]

File: src/panels/lovelace/components/hui-card-options.ts [hunk 2]
```diff
@@ -33,72 +33,79 @@ export class HuiCardOptions extends LitElement {
...
   static get styles(): CSSResult {
     return css`
-      div.options {
-        border-top: 1px solid #e8e8e8;
-        padding: 5px 8px;
-        background: var(--paper-card-background-color, white);
+      ha-card {
+        border-top-right-radius: 0;
+        border-top-left-radius: 0;
         box-shadow: rgba(0, 0, 0, 0.14) 0px 2px 2px 0px,
```
Comment (balloob) [inline src/panels/lovelace/components/hui-card-options.ts]: is this still needed ? Isn't this built into the card ?

**Comment by balloob** [comment]
ok to merge when last comment answered

File: src/panels/lovelace/components/hui-card-options.ts (hunk 2 above)
Comment (zsarnett) [reply src/panels/lovelace/components/hui-card-options.ts]: This is a different shadow from the Ha-card. It shifts the shadow down so there is no shadow on the overlapping section

**Pull request merged**